js/windowMgr.js

```diff
@@ -610,8 +602,19 @@ function doCreateMainWindow(initialUrl, initialBounds, isCustomTitleBar) {
         function devTools() {
             const focusedWindow = BrowserWindow.getFocusedWindow();
 
-            if (focusedWindow && !focusedWindow.isDestroyed()) {
-                focusedWindow.webContents.toggleDevTools();
+            if (devToolsEnabled) {
+                if (focusedWindow && !focusedWindow.isDestroyed()) {
```

Comment:
this condition can be moved to the above if statement

Reply:
@VishwasShashidhar Thanks for your comment. I fixed it